### PR TITLE
kubelet: Mount machine-id, resolv.conf

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: bootkube/resources/charts/kubelet/templates/kubelet-ds.yaml [hunk 1]
```diff
@@ -62,7 +62,6 @@ spec:
           readOnly: false
         - mountPath: /var/lib/calico
           name: coreos-var-lib-calico
-          readOnly: true
```
Comment (invidian) [inline bootkube/resources/charts/kubelet/templates/kubelet-ds.yaml]: Why is this needed?

**Review by invidian — DISMISSED** [review]

**Comment by surajssd** [comment]
baremetal tests failed so retriggering tests.

**Comment by invidian** [comment]
Pushed to trigger CI.

**Review by iaguis — DISMISSED** [review]

**Review by invidian — APPROVED** [review]
